packages/core/src/__integrationtests__/Ctypes.spec.ts

```diff
@@ -23,8 +23,17 @@ import { disconnect } from '../kilt'
 import { createEndowedTestAccount, initializeApi, submitTx } from './utils'
 
 let api: ApiPromise
+// we skip fetching CTypes from chain for the old pallet version, where the necessary information was not yet on chain.
+let hasBlockNumbers: boolean
 beforeAll(async () => {
   api = await initializeApi()
+  // this extrinsic should only exist in the new pallet version
```

Comment:
So you're not using the storage version value anymore?

Reply:
I'm happy to use it once it's fixed!

Reply:
Ok, then as a placeholder, this is the PR that bumps the CType storage version to 2 for the new type: https://github.com/KILTprotocol/kilt-node/pull/458. Once it's merged into develop and an image is built, it can be used.

Reply:
Will put this on hold then until https://github.com/KILTprotocol/kilt-node/pull/458 is merged.

Reply:
It has been. A new `latest-develop` tag is out.

Reply:
https://github.com/KILTprotocol/sdk-js/pull/703/commits/6fd18eb4f7f104cb4b21dda2ca42eaf0304c17e0